src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -115,7 +119,7 @@ public void createTimeline(final String eventTypeName, final String storageId, f
         }
         try {
             final EventType eventType = eventTypeCache.getEventType(eventTypeName);
-
+            authorizationValidator.authorizeEventTypeView(eventType);
```

Comment:
Why did you decide to check it for timeline creation endpoint? It's anyway an admin endpoint that allows access only to Nakadi-admins.

Reply:
As suggested, removed the check for both endpoints.